### PR TITLE
Bug/72

### PR DESCRIPTION
**Pull request by jaimecasero**
including new perf test and status code fix

### CODE REVIEW

File: core/slee/services/sbbs/src/main/java/org/mobicents/gmlc/slee/MobileCoreNetworkInterfaceSbb.java [hunk 1]
```diff
@@ -577,10 +577,10 @@ private void handleLocationResponse(MLPResponse.MLPResultType mlpResultType, CGI
 					getResponse.append(",vlrNumber=");
 					getResponse.append(response.vlr);
 
-                    this.sendHTTPResult(getResponse.toString());
+                    this.sendHTTPResult(HttpServletResponse.SC_OK, getResponse.toString());
                 }
                 else {
-                    this.sendHTTPResult(mlpClientErrorMessage);
+                    this.sendHTTPResult(HttpServletResponse.SC_INTERNAL_SERVER_ERROR, mlpClientErrorMessage);
```
Comment (jaimecasero) [inline core/slee/services/sbbs/src/main/java/org/mobicents/gmlc/slee/MobileCoreNetworkInterfaceSbb.java:583]: 500 error is quite generic, a per app error code translation should be implemented using proper http codes...,still 500 is better than nothing

Comment (FerUy) [inline core/slee/services/sbbs/src/main/java/org/mobicents/gmlc/slee/MobileCoreNetworkInterfaceSbb.java:583]: Agreed. Let's leave it like this for 1.0.0 milestone.

**Comment by FerUy** [comment]
Great as usual @jaimecasero, thanks!

**Comment by FerUy** [comment]
Just for the record, Restcomm-GMLC build 48 came out successfully with all tests OK :+1:  (including introduced HTTP Internal Server error code if applicable, e.g. when SS7 connection is out, HTTP error 501, "The server does not support the functionality needed to fulfill this request ()." is displayed).

**Pull request merged**